TaskWithMaxMinSortForArray/TaskWithMaxMinSortForArray.cpp

```diff
@@ -0,0 +1,59 @@
+﻿#include <iostream>
+
+int searchMax(int array[], size_t len) {
+	int max = array[0];
+	for (int i = 0; i < len; i++)
+	{
+		if (array[i] > max)
+		{
+			max = array[i];
+		}
+	}
+	return max;
+}
+
+int searchMin(int array[], size_t len) {
+	int min = array[0];
+	for (int i = 0; i < len; i++)
+	{
+		if (array[i] < min)
+		{
+			min = array[i];
+		}
+	}
+	return min;
+}
+
+void sortArray(int array[], size_t len) {
+	for (int i = 0; i < len - 1; i++) {
+		for (int j = 0; j < len - i - 1; j++) {
+			if (array[j] > array[j + 1]) {
+				int temp = array[j];
+				array[j] = array[j + 1];
+				array[j + 1] = temp;
+			}
+		}
+	}
+}
+
+int main(int argc, char* argv[])
+{
+	if (argc == 1)
+	{
+		std::cout << "No command line arguments " << std::endl;
+		return 1;
+	}
+	int* array = new int[argc - 1];
+	for (size_t i = 1; i < argc; i++)
+	{
+		array[i - 1] = std::atoi(argv[i]);
```

Comment:
What happens if I run your program with a cmd like?
`yourprogram.exe 1 2 3 four 5`

Reply:
The command line takes the string value as the digit 0 and outputs it as the minimum element of the array

Reply:
what if your string is not a number?

Reply:
The same. I checked this.